app/src/main/res/values/strings.xml

```diff
@@ -168,6 +168,8 @@
     <string name="search_history_deleted">Search history deleted.</string>
     <!-- error strings -->
     <string name="general_error">Error</string>
+    <string name="download_to_sdcard_error_title">External storage not available.</string>
+    <string name="download_to_sdcard_error_message">Download to external SD Card is not possible yet. Should the download place be reseted?</string>
```

Comment:
I think it should be _Should the download place be reset?_ as reset behaves like set and both are irregular.

Reply:
I changed it.